### ISSUE TITLE
Location Streaming confusing UX

### ISSUE THREAD

**Issue by AngryLasagna** [deltachat/deltachat-android#2185]
- Operating System (Linux/Mac/Windows/iOS/Android): Android 12 GrapheneOS
- Delta Chat Version: 1.26.2
- Expected behavior: 
Message location not recorded by default.
- Actual behavior:
Message locations are tagged by default without an off switch.
- Steps to reproduce the problem:
Try out Deltachat location streaming while bored in the train back home. Send messages meanwhile. Makes pink markers.
Next day get on train away from home, negotiating with a contact to meet via location tracking. Ex. Other person is in the train in the opposite direction and you want to know when to exit to meet at that station.
Now we both streamed location but I couldnt see the contacts marker. I didnt know what it would look like. Then at a station in the middle there was a lonely pink marker which I didnt remember was a message indtead of the other person. This made me exit the train unnecessarily.
TL;DR
Message pins are confusing if you arent used to it. Better make it an opt in toggle with explanation.
Also its metadata thats not always needed but neat to have explicitly.

- Screenshots:
No good ones.

- Logs:
Not relevant

**Comment by adbenitez** [comment]
I think it is fine how it is, peers' markers look the same way that you own markers while POIs are dots with text behind.

we can discuss this in the forum with all the user community: https://support.delta.chat/
